WordPress/Classes/ViewRelated/Reader/ReaderPostCardCell.swift

```diff
@@ -460,6 +566,14 @@ import WordPressShared
         blogNameLabel.highlighted = false
     }
 
+    @IBAction func tagButtonTouchesDidHighlight(sender: UIButton) {
+        tagLabelButton.highlighted = true
+    }
+
+    @IBAction func tagButtonTouchesDidEnd(sender: UIButton) {
+        tagLabelButton.highlighted = false
+    }
+
```

Comment:
Probably just need to swap "Touch Up Inside" to trigger `tagButtonTouchesDidEnd` instead of `tagButtonTouchesDidHighlight` to resolve https://github.com/wordpress-mobile/WordPress-iOS/pull/6188#issuecomment-260431117